src/serac/physics/heat_transfer.hpp

```diff
@@ -601,41 +604,68 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
    * @pre The adjoint load maps are expected to contain a single entry named "temperature"
    * @note If the essential boundary dual is not specified, homogeneous essential boundary conditions are applied to
    * the adjoint system
+   * @note We provide a quick derivation for the discrete adjoint equations and notation used here for backward
```

Comment:
It looks like this isn't quite rendering correctly in doxygen: https://serac.readthedocs.io/en/feature-tupek-transient_adj/doxygen/html/classserac_1_1HeatTransfer_3_01order_00_01dim_00_01Parameters_3_01parameter__space_8_8_8_01_4_00b10147e2187ea0fff3ed4d400f86c2eb.html

Reply:
I've enabled this branch in readhthedocs, so it should autobuild your new docs for each commit pushed.

Reply:
![image](https://github.com/LLNL/serac/assets/135926736/8061efff-f0c4-46c5-b5a8-cb5819113d82)

Reply:
here it is in the build path with doxygen

Reply:
Weird. I say we merge this and I'll investigate in another PR.